src/word_debt_bot/main.py

```diff
@@ -19,21 +19,14 @@ def get_token(token_path):
         return token_file.read()
 
 
-if __name__ == "__main__":
-    # Config
+def make_bot():
     intents = discord.Intents.default()
     intents.message_content = True
-    handler = logging.FileHandler(
-        filename=pathlib.Path("data/discord.log"), encoding="utf-8", mode="a"
-    )
-
-    # Loading data
-    game = WordDebtGame(pathlib.Path("data/prod.json"))
-    token = get_token(pathlib.Path("data/TOKEN"))
+    intents.members = True
```

Comment:
Adding members intent is breaking change, bot can't start unless app dev updates the intents on the apps page. Need to make a separate version for it to work in integration testing.